### PR TITLE
New Licenses: OpenSSL and SSLeay standalones

### LINKED ISSUE #2275: New license request: ssleay-windows [SPDX-Online-Tools]
**1.** License Name: Original SSLeay License with Windows Clause
**2.** Short identifier: ssleay-windows
**3.** License Author or steward: Unknown
**4.** Comments: This license is used in libgit2 package and has been discovered during License review in Fedora https://gitlab.com/fedora/legal/fedora-license-data/-/issues/425
This license is already recognized by scancode-toolkit https://scancode-licensedb.aboutcode.org/ssleay-windows
**5.** License Request Url: http://tools.spdx.org/app/license_requests/323
**6.** URL(s): https://github.com/libgit2/libgit2/blob/main/COPYING#L1030C1-L1086C36
**7.** OSI Status: Unknown
**8.** Example Projects: https://github.com/libgit2/libgit2/

### PR DESCRIPTION
**Pull request by jlovejoy**
fixes #2275 

This still needs some help on the Notes and not sure what URL to put for SSLeay-standalone?

Signed-off-by: Jilayne Lovejoy

### CODE REVIEW

**Comment by jlovejoy** [comment]
@richardfontana - do you have any thoughts on using the same URL for both "standalones" - or is there a URL we can use that shows just one or the other?

File: src/SSLeay-standalone.xml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="SSLeay-standalone" name="SSLeay License - standalone">
+      <crossRefs>
+         <crossRef>http://www.openssl.org/source/license.html</crossRef>
```
Comment (swinslow) [inline src/SSLeay-standalone.xml:5]: Found a cross-reference for this as a standalone: https://www.tq-group.com/filedownloads/files/software-license-conditions/OriginalSSLeay/OriginalSSLeay.pdf

File: src/SSLeay-standalone.xml [hunk 2]
```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="SSLeay-standalone" name="SSLeay License - standalone">
```
Comment (swinslow) [inline src/SSLeay-standalone.xml:3]: needs `listVersionAdded="3.23"`

File: src/OpenSSL-standalone.xml [hunk 3]
```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="OpenSSL-standalone" name="OpenSSL License - standalone">
```
Comment (swinslow) [inline src/OpenSSL-standalone.xml:3]: needs `listVersionAdded="3.23"`

File: src/OpenSSL-standalone.xml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+      <crossRefs>
+         <crossRef>http://www.openssl.org/source/license.html</crossRef>
```
Comment (swinslow) [inline src/OpenSSL-standalone.xml:5]: Found a couple of cross-references to this one as a standalone:

https://library.netapp.com/ecm/ecm_download_file/ECMP1196395

https://hstechdocs.helpsystems.com/manuals/globalscape/archive/cuteftp6/open_ssl_license_agreement.htm

File: src/OpenSSL-standalone.xml [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+      </crossRefs>
+      <notes>
+         This is the first license in the OpenSSL two-license stack as a standalone option.
+      </notes>
+      <text>
+         <titleText>
+            <p>
+               OpenSSL License (standalone)
```
Comment (swinslow) [inline src/OpenSSL-standalone.xml:13]: Do we see the license title appearing in the wild with "(standalone)" in it? If not, I would probably remove that. The notes and name/identifier help to identify it as standalone.

**Review by swinslow — COMMENTED** [review]
Hi @jlovejoy, I have a few nit edits and alternative cross-references to use. I'll submit a commit shortly to address these. Thanks!

**Comment by swinslow** [comment]
@jlovejoy Take a look at the updated version. After the tests pass, if the changes I made look good to you, feel free to merge. Thank you!

**Pull request merged**